app/models/user.rb

```diff
@@ -286,6 +286,10 @@ def warned
     has_role? :warned
   end
 
+  def banished?
+    return true if user.notes.where(reason: "banned", content: "spam account").any? && user.banned
```

Comment:
I think this could be just `user.notes.where(reason: "banned", content: "spam account").any? && user.banned` w/o explicit return.

Reply:
Yeah implicit return is the cleaner Ruby way to go